test/integration/stats_integration_test.cc

```diff
@@ -369,6 +373,7 @@ TEST_P(ClusterMemoryTestRunner, MemoryLargeHostSizeWithStats) {
   // 2019/11/01  8859     1299         1315   build: switch to libc++ by default
   // 2019/11/12  8998     1299         1350   test: adjust memory limit for macOS
   // 2019/11/15  9040     1283         1350   build: update protobuf to 3.10.1
+  // 2020/01/13  9663     1619         1619   api: deprecate hosts in Cluster.
```

Comment:
This is a 25% regression, which seems scary. This occurred before I merged #9633, so I don't think it has to do with original type information. Best guess is that the deeper structure results in a bunch of mallocs, each with some proto book keeping.